bootstrap.php

```diff
@@ -244,16 +244,15 @@
     ->addCall('addEmbed', [new Reference(Vanilla\Embeds\TwitterEmbed::class)])
     ->addCall('addEmbed', [new Reference(Vanilla\Embeds\YouTubeEmbed::class)])
     ->addCall('addEmbed', [new Reference(Vanilla\Embeds\VimeoEmbed::class)])
-    ->addCall('addEmbed', [new Reference(Vanilla\Embeds\InstagramEmbed::class)])
+    ->addCall('addEmbed', [new Reference(Vanilla\Embeds\ImgurEmbed::class)])
     ->addCall('addEmbed', [new Reference(Vanilla\Embeds\SoundCloudEmbed::class)])
     ->addCall('addEmbed', [new Reference(Vanilla\Embeds\ImageEmbed::class), Vanilla\Embeds\EmbedManager::PRIORITY_LOW])
     ->setShared(true)
 
     ->rule(Vanilla\PageScraper::class)
     ->addCall('registerMetadataParser', [new Reference(Vanilla\Metadata\Parser\OpenGraphParser::class)])
     ->addCall('registerMetadataParser', [new Reference(Vanilla\Metadata\Parser\JsonLDParser::class)])
-    ->setShared(true)
-;
+    ->setShared(true);
```

Comment:
Why'd you move the semicolon? It's better to keep it separate in large chained calls, like this.